src/app/fyle/view-per-diem/view-per-diem.page.spec.ts

```
@@ -273,14 +273,15 @@ describe('ViewPerDiemPage', () => {
   });
 
   describe('ionViewWillEnter():', () => {
+    const mockCustomFields = cloneDeep(customFields);
```

cloned the mock data as it was causing race condition (other tests were failing due to change in mock data)